test/test_rpc.ts

```diff
@@ -20,4 +29,15 @@ describe("ts-rpc", () => {
     assert(stub);
     assert.equal(await stub.add(4, 5), 9);
   });
+
+  it("should support hello world without a checker", async () => {
+    const aRpc = new Rpc();
+    const bRpc = new Rpc();
+    aRpc.start((msg) => bRpc.receiveMessage(msg));
+    bRpc.start((msg) => aRpc.receiveMessage(msg));
+    aRpc.registerImpl("my-greeting", new MyGreeting());
```

Comment:
I'd like to move to what the Rpc module documentation proposes, which is to pass in `rpc.unchecked` for the required checker argument. The idea is to make it easy to skip checks if that's actually desired, but hard to omit checking by mistake.

Reply:
I've created an issue for it #3